src/partials/index_scripts.html

```diff
@@ -1 +1,6 @@
-<script src="./js/index.js" type="module"></script>
\ No newline at end of file
+<script src="./js/index.js" type="module"></script>
+<script defer src="https://cdnjs.cloudflare.com/ajax/libs/body-scroll-lock/3.1.5/bodyScrollLock.min.js"
```

Comment:
lets setup body-scroll-lock using npm

Reply:
npm i body-scroll-lock
https://www.npmjs.com/package/body-scroll-lock#reactes6 for example

Reply:
Done. Replaced with npm realization.